dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05210CreateDefaultDotAsset.java

```diff
@@ -0,0 +1,205 @@
+     * with them.
+     * 
+     * @return The List of DotAsset Variable fields.
+     */
+    @SuppressWarnings("unchecked")
+    private List<Map<String, Object>> getDotAssetVariableFields() {
+
+        final boolean isRequired   = Boolean.TRUE;
+        final boolean noRequired   = Boolean.FALSE;
+        final boolean noListed     = Boolean.FALSE;
+        final boolean isIndexed    = Boolean.TRUE;
+        final boolean isFixed      = Boolean.TRUE;
+        final boolean noFixed      = Boolean.FALSE;
+        final boolean noReadOnly   = Boolean.FALSE;
+        final boolean noUnique     = Boolean.FALSE;
+        final boolean isSearchable = Boolean.TRUE;
+
+        // hostFolder
+        final Map<String, Object> hostFolderField = setFieldElements("Site or Folder",
+                HostFolderField.class.getName(), StringPool.BLANK, "system_field", noRequired, noListed, isIndexed,
+                DotAssetContentType.SITE_OR_FOLDER_FIELD_VAR, 0, StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+                StringPool.BLANK, noFixed, noReadOnly, isSearchable, noUnique);
+
+        // Asset
+        final Map<String, Object> assetField = setFieldElements("Binary Asset",
+                BinaryField.class.getName(), StringPool.BLANK, "system_field", isRequired, noListed, isIndexed,
+                DotAssetContentType.ASSET_FIELD_VAR, 1, StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+                StringPool.BLANK, isFixed, noReadOnly, isSearchable, noUnique);
+
+        // tags
+        final Map<String, Object> tagsField = setFieldElements("Tags",
+                TagField.class.getName(), StringPool.BLANK, "system_field", noRequired, noListed, isIndexed,
+                DotAssetContentType.TAGS_FIELD_VAR, 2, StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+                StringPool.BLANK, noFixed, noReadOnly, isSearchable, noUnique);
+
+        return list(assetField, hostFolderField, tagsField);
+    }
+
+    /**
+     * This method sets the value of each element that is needed when creating a Field.
+     * 
+     * @param fieldName
+     * @param fieldType
+     * @param fieldRelationType
+     * @param fieldContentlet
+     * @param required
+     * @param listed
+     * @param indexed
+     * @param velocityVarName
+     * @param sortOrder
+     * @param values
+     * @param checkRegex
+     * @param hint
+     * @param defaultValue
+     * @param fixed
+     * @param readOnly
+     * @param searchable
+     * @param unique
+     * @return The data required by a field.
+     */
+    private Map<String, Object> setFieldElements(String fieldName, String fieldType, String fieldRelationType,
+                    String fieldContentlet, boolean required, boolean listed, boolean indexed, String velocityVarName,
+                    int sortOrder, String values, String checkRegex, String hint, String defaultValue, boolean fixed,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'values' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4995431)